packages/babel-plugin-transform-charcodes/src/index.js

```diff
@@ -1,5 +1,3 @@
-// @flow
```

Comment:
@existentialism why did you remove the comment?

Reply:
@Andarist just temporary, mostly because it had a ton of flow errors, and it'd be primarily making everything `any`.